Makefile

```diff
@@ -182,6 +182,10 @@ e2e-test:  $(ALL_SOURCE_FILES) all-binaries
 vpc-branch-eni-e2e-tests: $(ALL_SOURCE_FILES) vpc-branch-eni
 	sudo -E CNI_PATH=$(CUR_DIR)/$(BUILD_DIR) go test -v -tags e2e_test -race -timeout 60s ./plugins/vpc-branch-eni/e2eTests/
 
+.PHONY: vpc-eni-e2e-tests
+vpc-eni-e2e-tests: $(ALL_SOURCE_FILES) vpc-eni
+	sudo -E CNI_PATH=$(CUR_DIR)/$(BUILD_DIR) go test -v -tags e2e_test -race -timeout 60s ./plugins/vpc-eni/e2eTests/
+
```

Comment:
We have an unofficial rule to operate on the plugins in the same order as the list in line 62. Therefore I would add this new block for vpc-eni just before that of vpc-branch-eni.

Reply:
Sure, I will make the change.

Reply:
Updated in [ed8da834](https://github.com/aws/amazon-vpc-cni-plugins/pull/103/commits/ed8da834c06a5606b85bd5dc8ef6b8e93323c9fc).